python/salpytools/salpylib.py

```diff
@@ -141,8 +162,8 @@ def send_logEvent(self, eventname, **kwargs):
             try:
                 self.myData[eventname].settings = self.settings
             except Exception:
-                LOGGER.info(
-                    "WARNING: Could not extract 'settings' from state to reply the 'settingsApplied'")
+                msg = "WARNING: Could not extract 'settings' from state to reply the 'settingsApplied'"
+                LOGGER.warning(msg)
```

Comment:
We can use python `logging` because there is a feature of `lsst.log` that can intercept `logging`.  Using `logging` is fine because you can use higher level code to decide where the logs end up.